block/src/raw_async_aio.rs

```diff
@@ -0,0 +1,138 @@
+use crate::async_io::{
```

Comment:
This new file hasn't got a copyright header.

Reply:
Thanks for looking back at this @rbradford. Put out a PR to add.

https://github.com/cloud-hypervisor/cloud-hypervisor/pull/5888